ehcache-core/src/main/java/org/ehcache/core/EhcachePrefixLoggerFactory.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.core;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+public class EhcachePrefixLoggerFactory {
+
+  private EhcachePrefixLoggerFactory() {
+    throw new UnsupportedOperationException("This is a utility class and cannot be instantiated");
+  }
+
+  private static final ThreadLocal<Map<String, String>> bindTimeMdc = new ThreadLocal<>();
+
+  public static Map<String, String> getContextMap() {
```

Comment:
This should be package protected to hide it as much as possible. We should probably also add a comment to indicate it's only here for test purposes.

Reply:
changed the access to protected